README.md

```diff
@@ -9,17 +9,42 @@ nio
 
 
 
-nio is a multilayered matrix client library. The underlying base layer doesn't
-do any IO on its own. On top of the base layer, a no-IO HTTP client
-implementation exists, as well as a full fledged batteries included asyncio
-layer using [aiohttp](https://github.com/aio-libs/aiohttp/).
+nio is a multilayered [Matrix](https://matrix.org/) client library. The
+underlying base layer doesn't do any IO on its own. On top of the base layer, a
+no-IO HTTP client implementation exists, as well as a full fledged batteries
```

Comment:
I think we should de-emphasize the existence of the no-IO HTTP client since It's way to hard to use,  nobody should use it, and I plan on deprecating it.